var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -101,7 +102,7 @@ class Abinit(AutotoolsPackage):
     # using `conflicts`, `depends_on` could be used instead.
     mkl_message = 'Need to set dependent variant to threads=openmp'
     conflicts('+openmp',
-              when='^fftw~openmp',
+              when='^fftw-api~openmp',
```

Comment:
There is no defined interface for `fftw-api`, so you can't assume you can set `~openmp` on it. You'll need to do that per implementation now.

Reply:
fixed.

Reply:
@haampie
Do I need to make other changes?